src/common/store/gameFormats.js

```diff
@@ -1,10 +1,19 @@
 import { cloneDeep, shuffle } from 'lodash';
 import * as seededRNG from 'seed-random';
```

Comment:
This file should be ts since it's multiplayer stuff right?

Reply:
It pertains to multiplayer but it's `src/common` (as opposed to `src/server`) so it doesn't _have_ to be TS.

That said, I'll probably convert it to TS in a separate PR later.